client/components/domains/connect-domain-step/page-definitions.js

```diff
@@ -103,6 +107,44 @@ export const connectADomainOwnershipVerificationStepsDefinition = {
 	},
 };
 
+export const transferDomainStepsDefinition = {
+	[ stepSlug.TRANSFER_START ]: {
+		mode: modeType.TRANSFER,
+		step: stepType.START,
+		component: TransferDomainStepStart,
+		next: stepSlug.TRANSFER_LOGIN,
+	},
+	[ stepSlug.TRANSFER_LOGIN ]: {
+		mode: modeType.TRANSFER,
+		step: stepType.LOG_IN_TO_PROVIDER,
+		name: __( 'Log in to provider' ),
+		component: TransferDomainStepLogin,
+		next: stepSlug.TRANSFER_UNLOCK,
+		prev: stepSlug.TRANSFER_START,
+	},
+	[ stepSlug.TRANSFER_UNLOCK ]: {
+		mode: modeType.TRANSFER,
+		step: stepType.UNLOCK_DOMAIN,
+		name: __( 'Unlock domain' ),
+		component: TransferDomainStepUnlock,
+		next: stepSlug.TRANSFER_AUTH_CODE,
+		prev: stepSlug.TRANSFER_LOGIN,
+	},
+	[ stepSlug.TRANSFER_AUTH_CODE ]: {
+		mode: modeType.TRANSFER,
+		step: stepType.ENTER_AUTH_CODE,
+		name: __( 'Get domain authorization code' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Get a domain authorization code.' )` *ES Score: 10*